Projeto Físico/semijoin.sql

```diff
@@ -0,0 +1,12 @@
+SELECT F.NOME
+FROM FUNCIONARIO F
+WHERE F.MENTOR IS NOT NULL AND F.CPF IN (
+    SELECT T.CPF
+    FROM TRABALHA T
+    WHERE T.CODIGO IN (
+        SELECT T.CODIGO
+        FROM TRABALHA T
+        WHERE T.CPF = F.MENTOR)
```

Comment:
Eu não sei se dá erro de lógica, porque de sintaxe não dá, mas não seria melhor diferenciar os Ts?
```suggestion
    SELECT T1.CPF
    FROM TRABALHA T1
    WHERE T1.CODIGO IN (
        SELECT T2.CODIGO
        FROM TRABALHA T2
        WHERE T2.CPF = F.MENTOR)
)
```